Common/ContextMenu.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace Files.Common
+{
+    // Same definition of Vanara.PInvoke.User32.MenuItemType
+    public enum MenuItemType : uint
+    {
+        MFT_STRING = 0,
+        MFT_BITMAP = 4,
+        MFT_MENUBARBREAK = 32,
+        MFT_MENUBREAK = 64,
+        MFT_OWNERDRAW = 256,
+        MFT_RADIOCHECK = 512,
+        MFT_SEPARATOR = 2048,
+        MFT_RIGHTORDER = 8192,
+        MFT_RIGHTJUSTIFY = 16384
+    }
+
+    public class IWin32ContextMenu
```

Comment:
Why class name starts with 'I'? It is not an interface. Same for `IWin32ContextMenuItem`

Reply:
Right, it used to be